test/integration/file-discovery.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	swarm_manage_cleanup
+	rm $DISCOVERY_FILE
+	stop_docker
+}
+
+function setup_file_discovery() {
+	# create a temp file
+	DISCOVERY_FILE=$(mktemp)
```

Comment:
**Maybe** it's better to put this outside of the function.

If the test fails before calling this (`start_docker` fails), `DISCOVERY_FILE` will be undefined and the teardown will `rm` with no arguments.

Not a huge deal.